README.md

```diff
@@ -395,9 +395,14 @@ Add a token transfer to the `data.ledger.token.transfers` section of the transac
 
 #### addRecipient(to)
 
-Add a recipient (for non UCO transfers, ie. smart contract interaction) to the `data.recipient` section of the transaction
-
-- `to` is hexadecimal encoding or Uint8Array representing the transaction address (recipient)
+ Adds a recipient to call the smart contract's "transaction" action.
```

Comment:
The verb "Add" is more direct and imperative, which is often used in instructional or informational documentation. The first version "Adds" might imply a descriptive stance, as if it's describing a function that automatically adds a recipient. On the other hand, "Add" implies a directive to the reader or developer, instructing them to perform an action.